docs/API_Reference_Guide.rst

```diff
@@ -70,93 +69,93 @@ hiptensorContractionPlan_t
 Helper Functions
 ================
 
-hiptensorGetVersion
+:code:`hiptensorGetVersion`
 -------------------
 
 .. doxygenfunction::  hiptensorGetVersion
 
-hiptensorCreate
+:code:`hiptensorCreate`
 ---------------
 
 .. doxygenfunction::  hiptensorCreate
 
-hiptensorDestroy
+:code:`hiptensorDestroy`
 ----------------
 
 .. doxygenfunction::  hiptensorDestroy
 
-hiptensorInitTensorDescriptor
+:code:`hiptensorInitTensorDescriptor`
 -----------------------------
 
 .. doxygenfunction::  hiptensorInitTensorDescriptor
 
-hiptensorGetAlignmentRequirement
+:code:`hiptensorGetAlignmentRequirement`
 --------------------------------
 
 .. doxygenfunction::  hiptensorGetAlignmentRequirement
 
-hiptensorGetErrorString
+:code:`hiptensorGetErrorString`
 -----------------------
 
 .. doxygenfunction::  hiptensorGetErrorString
 
 Contraction Operations
 ======================
 
-hiptensorInitContractionDescriptor
+:code:`hiptensorInitContractionDescriptor`
 ----------------------------------
 
 .. doxygenfunction::  hiptensorInitContractionDescriptor
 
-hiptensorInitContractionFind
+:code:`hiptensorInitContractionFind`
 ----------------------------
 
 .. doxygenfunction::  hiptensorInitContractionFind
 
-hiptensorInitContractionPlan
+:code:`hiptensorInitContractionPlan`
 ----------------------------
 
 .. doxygenfunction::  hiptensorInitContractionPlan
 
-hiptensorContraction
+:code:`hiptensorContraction`
 --------------------
 
 .. doxygenfunction::  hiptensorContraction
 
-hiptensorContractionGetWorkspaceSize
+:code:`hiptensorContractionGetWorkspaceSize`
 ------------------------------------
 
 .. doxygenfunction::  hiptensorContractionGetWorkspaceSize
 
 Logging Functions
 =================
 
-hiptensorLoggerSetCallback
+:code:`hiptensorLoggerSetCallback`
 --------------------------
 
 .. doxygenfunction::  hiptensorLoggerSetCallback
 
-hiptensorLoggerSetFile
+:code:`hiptensorLoggerSetFile`
 ----------------------
 
 .. doxygenfunction::  hiptensorLoggerSetFile
 
-hiptensorLoggerOpenFile
+:code:`hiptensorLoggerOpenFile`
 -----------------------
 
 .. doxygenfunction::  hiptensorLoggerOpenFile
 
-hiptensorLoggerSetLevel
+:code:`hiptensorLoggerSetLevel`
 -----------------------
 
 .. doxygenfunction::  hiptensorLoggerSetLevel
 
-hiptensorLoggerSetMask
+:code:`hiptensorLoggerSetMask`
 ----------------------
 
 .. doxygenfunction::  hiptensorLoggerSetMask
 
-hiptensorLoggerForceDisable
+:code:`hiptensorLoggerForceDisable`
```

Comment:
I would revert these.

Reply:
[Revert "Use code directive to escape code keywords"](https://github.com/ROCmSoftwarePlatform/hipTensor/pull/155/commits/7b8c448f979cd02350184e31ca7a6667d161329d) 
[7b8c448](https://github.com/ROCmSoftwarePlatform/hipTensor/pull/155/commits/7b8c448f979cd02350184e31ca7a6667d161329d)
This reverts commit https://github.com/samjwu/hipTensor/commit/7be7e3446830f2d7dd2760f140c7dfa30a246f78.